Build/Remotion.Local.build

```diff
@@ -40,20 +45,22 @@
   <Target Name="TestBuild_Configuration">
     <PropertyGroup>
       <Platforms>x86</Platforms>
-      <DatabaseSystems>SqlServerDefault</DatabaseSystems>
-      <Browsers></Browsers>
+      <DatabaseSystems>NoDB+SqlServerDefault</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>LocalMachine</ExecutionRuntimes>
+      <TargetRuntimes>NET48</TargetRuntimes>
       <TestCategoriesToExclude></TestCategoriesToExclude>
       <SkipTests>False</SkipTests>
     </PropertyGroup>
   </Target>
   
   <Target Name="FullBuild_Configuration">
     <PropertyGroup>
-      <!-- Skip x86 -->
       <Platforms>x64</Platforms>
-      <DatabaseSystems>SqlServerDefault</DatabaseSystems>
-      <!-- Skip Internet Explorer -->
-      <Browsers>Chrome</Browsers>
+      <DatabaseSystems>NoDB+SqlServerDefault</DatabaseSystems>
+      <Browsers>NoBrowser+Chrome</Browsers>
```

Comment:
Same thing here, I think we should only do NoDB and NoBrowser. In the re-motion trunk in SecurityManager, we did the same thing so I think that's what MK would want